app.js

```diff
@@ -7,9 +7,9 @@ const randomInt = (min, max) => Math.floor(Math.random() * (max - min) + 1);
 const randomColor = () =>
   `rgb(${randomInt(0, 255)}, ${randomInt(0, 255)}, ${randomInt(0, 255)})`;
 
-const initialBox = colorBox.style.visibility = 'hidden';
+const initialBox = (colorBox.style.visibility = "hidden");
```

Comment:
what is the difference in writing inside a round bracket?
@Abbas4tech

Reply:
round brakcets let JS know that its an expression that needs to resolve first and store in a `const` value